src/main/java/com/ot/restaurant/controllers/CustomerController.java

```
@@ -0,0 +1,34 @@
+package com.ot.restaurant.controllers;
+
+import com.ot.restaurant.entities.Customer;
+import com.ot.restaurant.repositories.CustomerRepository;
+import constants.Status;
+import java.util.List;
+
+public class CustomerController {
+  private CustomerRepository customerRepository;
+
+  public CustomerController(CustomerRepository customerRepository) {
+    this.customerRepository = customerRepository;
+  }
+
+  public List<Customer> findAll() throws Exception {
```

Do you really need to throw an exception for all the methods?

---

Yes, because we are throwing an exception in the interface, and the IDE asks to handle the exception.